src/components/CheckboxInput2/CheckboxInput2.js

```diff
@@ -118,6 +119,7 @@ export const CheckboxInput2 = ({
               type="checkbox"
               onChange={onChange}
               data-tid={rest['data-tid']}
+              data-formotiv={dataFormotiv}
```

Comment:
minor: Would it be possible to only include this attribute when `dataFormotiv !== ''`?

Reply:
That might be hard to do without creating an object of `props` and then conditionally using `Object.assign(props, {data-formotiv: dataFormotiv}`

Reply:
Yeah fair enough, it's not a big deal to leave as is.